devtools/conda-envs/test_env.yaml

```diff
@@ -23,7 +23,8 @@ dependencies:
   - isort
   - black >=22.1.0
   - flake8
-  
+  - versioneer
```

Comment:
(not blocking merging) It's likely that this doesn't actually need to be here

Reply:
> (not blocking merging) It's likely that this doesn't actually need to be here

You're right, versioneer was only needed once (to regen `paprika/_version.py`). I'll take it out of here, thanks!